server/game/cards/01_SOR/units/StarWingScout.ts

```diff
@@ -0,0 +1,25 @@
+import AbilityHelper from '../../../AbilityHelper';
```

Comment:
Just a note - in the future please don't add new cards to an already open PR. New code probably means more rounds of review so the PR will kind of drag on for a while, and also it can make it harder to know what's been reviewed and what's new.

Folks doing lots of card implementations typically batch a handful of cards into a PR and then move on to do a new batch while they wait for review, that's a good pattern to follow.

Reply:
Understood.